frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -45,10 +45,10 @@ function getDesignSystemCssVariables(theme: MantineTheme) {
     --mb-color-bg-dark: ${theme.fn.themeColor("bg-dark")};
     --mb-color-brand: ${theme.fn.themeColor("brand")};
 
-    --mb-color-brand-light: color-mix(in srgb, var(--mb-color-brand) 53%, #fff);
+    --mb-color-brand-light: color-mix(in srgb, var(--mb-color-brand) 4%, #fff);
```

Comment:
The changes are eye-balled. I tested with loki by generating snapshots with previous colors and new colors and they passed without showing any diff.